### ISSUE TITLE
Puppetforge nodejs is not the same version as on github

### ISSUE THREAD

**Issue by sebastianwahn** [voxpupuli/puppet-nodejs#182]
If i download the puppetforge  [puppet/nodejs](https://forge.puppetlabs.com/puppet/nodejs) via the tar.gz download, i get a version which says 1.2.0 in the changelog.

But if i check the file "manifests/init.pp" it is not the same file as in this github repository. 

If i download the old(?) [puppetlabs/nodejs](https://forge.puppetlabs.com/puppetlabs/nodejs) version which says 0.8.0 in the changelog, the file "manifests/init.pp" is the same as the 1.2.0 version of puppet/nodejs which i downloaded previously.

In other words: [puppet/nodejs](https://forge.puppetlabs.com/puppet/nodejs) as version 1.2.0 and [puppetlabs/nodejs](https://forge.puppetlabs.com/puppetlabs/nodejs) as version 0.8.0 are (nearly?) the same files when downloaded via tar.gz (maybe also via `puppet module`).

Is there a problem in your CI or whatever you use to distribute the module?

**Comment by igalic** [comment]
this def needs a new release

**Comment by ColinHebert** [comment]
When will it be possible to get a new releasE?

**Comment by ColinHebert** [comment]
@igalic Would you mind releasing a new version (and upload it to the puppet forge?)
